velox/substrait/tests/VeloxSubstraitRoundTripTest.cpp

```diff
@@ -313,6 +311,38 @@ TEST_F(VeloxSubstraitRoundTripTest, notNullLiteral) {
       plan, "SELECT true, 23, 45, 678, 910, 1.23, 4.56, '789'");
 }
 
+namespace {
+core::TypedExprPtr makeConstantExpr(const VectorPtr& vector) {
+  return std::make_shared<const core::ConstantTypedExpr>(
+      BaseVector::wrapInConstant(1, 0, vector));
+}
+} // namespace
+
+TEST_F(VeloxSubstraitRoundTripTest, arrayLiteral) {
+  auto vectors = makeRowVector(ROW({}), 1);
+  auto plan =
+      PlanBuilder(pool_.get())
+          .values({vectors})
+          .addNode([&](std::string id, core::PlanNodePtr input) {
+            std::vector<std::string> names = {"a", "b", "c"};
+            std::vector<core::TypedExprPtr> expressions = {
+                // [1, null]
+                makeConstantExpr(
+                    makeNullableArrayVector<int64_t>({{1, std::nullopt}})),
```

Comment:
The code paths are different for different types. This test covers only BIGINT type. Would you add tests for other types?

Reply:
just added all types.